webpack/webpack.config.dll.js

```diff
@@ -8,7 +8,7 @@ module.exports = {
       '@babel/polyfill',
       // '@clusterws/cws',
       // '@google-cloud/language',
-      '@mattkrick/fast-rtc-peer',
+      // '@mattkrick/fast-rtc-peer', can put back in after RTC branch is merged
```

Comment:
It's now 100% `fast-rtc-swarm` tho, right?

Reply:
yeah, i can probably remove peer or at least bump it to a dev dependency